requirements.in

```diff
@@ -30,3 +30,4 @@ sentry_sdk
 whitenoise
 social-auth-app-django>=5.0.0
 python-pptx
+pyyaml>=6
```

Comment:
This line is not actually required and I think it shouldn't be here unless it's a core requirement used in our own code. It helps when updating the requirement, but to bump a requirement you can also just run `pip-compile -P pyyaml requirements.in`